### PR TITLE
#415/banned users shouldn't be able to use any endpoint

### PR DESCRIPTION
**Pull request by Damrod**
as explaine in #415

### CODE REVIEW

File: transcendence-app/src/chat/chat.gateway.ts [hunk 1]
```diff
@@ -132,6 +132,9 @@ export class ChatGateway {
       client.request.user.id,
       chatroomId,
     );
+    if (!crm) {
```
Comment (Damrod) [inline transcendence-app/src/chat/chat.gateway.ts:135]: before, there was some code path where we could be throwing an http exception here. Now it should happen, even conceptually, anymore. I'm not sure what would happen if we tried http exceptions in ws tbh.

**Comment by Damrod** [comment]
I think this part about the backend of the ban user feature is mostly ready. We could try and implement a ws event that logs out some user in real time after any admin bans them. I'm looking into it, but it's mostly a plus.

File: transcendence-app/src/authorization/authorization.service.ts [hunk 2]
```diff
@@ -103,18 +103,22 @@ export class AuthorizationService {
...
+        ...g_user,
+        chatId,
+        crm_member: !!crm,
+        crm_owner: crm?.owner,
+        crm_admin: crm?.admin,
+        crm_banned: crm?.banned,
+        cr_muted: crm?.muted,
+      });
+    } catch (e) {
+      return null;
```
Comment (vdedios) [inline transcendence-app/src/authorization/authorization.service.ts:120]: what about logging this error?

**Comment by vdedios** [comment]
> I think this part about the backend of the ban user feature is mostly ready. We could try and implement a ws event that logs out some user in real time after any admin bans them. I'm looking into it, but it's mostly a plus.

Although if would be nice, if the user already cannot navigate else where once is banned, I think we are at a very good point right now with this 👍

File: transcendence-app/src/authorization/authorization.service.ts (hunk 2 above)
Comment (Damrod) [reply transcendence-app/src/authorization/authorization.service.ts:120]: done

**Pull request merged**